backend/qemu.pm

```diff
@@ -545,16 +545,20 @@ sub start_qemu {
                 # XXX: workaround for OVMF wanting to write NVvars into first FAT partition
                 # we need to replace -bios with proper pflash drive specification
                 $params[-1] .= ',snapshot=on' if $vars->{UEFI};
-                push(@params, '-device', "$vars->{CDMODEL},drive=cd0,bus=scsi0.0");
+                my $bus //= '';
```

Comment:
I don't get why are you using `//` operator. The `$bus` will be always undef this time so you can just do plain initialization `my $bus = '';`

Or fancy `my $bus = $vars->{CDMODEL} ne 'ide-cd' ? ',bus=scsi0.0' : '';`